can/notifier.py

```diff
@@ -138,6 +142,10 @@ def _on_message_received(self, msg: Message) -> None:
                 self._loop.create_task(res)
 
     def _on_error(self, exc: Exception) -> bool:
+        """Calls ``on_error()`` for all listeners if they implement it.
+
+        :returns: ``True`` if and only if at least one handler was called
+        """
         listeners_with_on_error = [
             listener for listener in self.listeners if hasattr(listener, "on_error")
```

Comment:
Isn't this the actual root cause? We check whether `on_error` exists, but the base class `Listener` already defined it. So every listenere has that method whether it handles the exception or not.

I'd suggest to raise NotImplementedError in the base class and call the method for every listener here.

Reply:
Yeah, I guess. Sounds reasonable. I'll adjust this PR real quick, then you can give some feedback.

Reply:
Looks good to me. Thank you. But i really hope CI works again soon :D